rules/sonic-utilities.mk

```diff
@@ -2,4 +2,5 @@
 
 SONIC_UTILS = python-sonic-utilities_1.1-1_all.deb
 $(SONIC_UTILS)_SRC_PATH = $(SRC_PATH)/sonic-utilities
+$(SONIC_UTILS)_WHEEL_DEPENDS = $(SONIC_CONFIG_ENGINE)
```

Comment:
Can you also add dependency to swsssdk?

Reply:
swsssdk is the config-engine dependency, no need to explicit specify that. Check the build log below.

```
lgh@ba2755a9022e:/data/sonic/sonic-buildimage$ make -f slave.mk  target/debs/python-sonic-utilities_1.1-1_all.deb        
SONiC Build System

Build Configuration
"CONFIGURED_PLATFORM"             : "vs"
"SONIC_CONFIG_PRINT_DEPENDENCIES" : ""
"SONIC_CONFIG_BUILD_JOBS"         : "1"
"SONIC_CONFIG_MAKE_JOBS"          : "4"
"DEFAULT_USERNAME"                : "admin"
"DEFAULT_PASSWORD"                : "YourPaSsWoRd"
"ENABLE_DHCP_GRAPH_SERVICE"       : ""
"SHUTDOWN_BGP_ON_START"           : ""
"SONIC_CONFIG_DEBUG"              : ""
"ROUTING_STACK"                   : "quagga"
"ENABLE_SYNCD_RPC"                : ""
"ENABLE_ORGANIZATION_EXTENSIONS"  : "y"

[ building ] [ target/python-wheels/swsssdk-2.0.1-py2-none-any.whl ] 
[ finished ] [ target/python-wheels/swsssdk-2.0.1-py2-none-any.whl ] 
[ building ] [ target/python-wheels/swsssdk-2.0.1-py2-none-any.whl-install ] 
[ finished ] [ target/python-wheels/swsssdk-2.0.1-py2-none-any.whl-install ] 
[ building ] [ target/python-wheels/sonic_config_engine-1.0-py2-none-any.whl ] 
[ finished ] [ target/python-wheels/sonic_config_engine-1.0-py2-none-any.whl ] 
[ building ] [ target/python-wheels/sonic_config_engine-1.0-py2-none-any.whl-install ] 
[ finished ] [ target/python-wheels/sonic_config_engine-1.0-py2-none-any.whl-install ] 
[ building ] [ target/debs/python-sonic-utilities_1.1-1_all.deb ] 
[ finished ] [ target/debs/python-sonic-utilities_1.1-1_all.deb ] 
```